enaml/qt/docking/__init__.py

```diff
@@ -1,7 +1,14 @@
 #------------------------------------------------------------------------------
-# Copyright (c) 2013-2017, Nucleic Development Team.
+# Copyright (c) 2013-2022, Nucleic Development Team.
 #
 # Distributed under the terms of the Modified BSD License.
 #
 # The full license is in the file LICENSE, distributed with this software.
 #------------------------------------------------------------------------------
+from .. import QT_VERSION
+
+
+def hover_event_pos(event):
+    if QT_VERSION[0] == "6":
```

Comment:
Instead of checking the version in every event it'd be slightly better to define a different function

```python
if QT_VERSION[0] == "6":
    def hover_event_pos(event):
        return event.position().toPoint()
else:
    def hover_event_pos(event):
        return event.pos()

```

Reply:
I wanted to suggest the same thing but then confused myself about switching the qt version at runtime...